scvi/models/vae.py

```diff
@@ -36,17 +44,19 @@ class VAE(nn.Module):
 
     """
 
-    def __init__(self, n_input, n_batch=0, n_labels=0, n_hidden=128, n_latent=10, n_layers=1, dropout_rate=0.1,
-                 dispersion="gene", log_variational=True, reconstruction_loss="zinb"):
+    def __init__(self, n_input: int, n_batch: int = 0, n_labels: int = 0,
+                 n_hidden: int = 128, n_latent: int = 10, n_layers: int = 1,
+                 dropout_rate: float = 0.1, dispersion: str = "gene",
+                 log_variational: bool = True, reconstruction_loss: str = "zinb"):
         super(VAE, self).__init__()
         self.dispersion = dispersion
         self.n_latent = n_latent
         self.log_variational = log_variational
         self.reconstruction_loss = reconstruction_loss
-        # Automatically desactivate if useless
+        # Automatically deactivate if useless
         self.n_batch = n_batch
         self.n_labels = n_labels
-        self.n_latent_layers = 1
+        self.n_latent_layers = 1  # not sure what this is for, no usages?
```

Comment:
Should it be set to the value of `n_layers` then? As I commented, it doesn't appear to be used in the code anywhere, and it's always set to 1 (or 2 in `SVAEC`), so I wasn't sure what it was for.

Reply:
Uh, it may be different than `n_layers` by 1. I'm not sure. @Edouard360 do you know what it's called `n_latent_layers` and not `n_layers`?

Reply:
`n_latent_layers = 1`  for `VAE` and `n_latent_layers = 2` for `SVAEC`.
It's the number of levels for the latent stochastic variables.
At one point we had ladder autoencoder models as well, which had multiple levels of stochastic variables. We no longer have the corresponding visualization tools on `master` so perhaps it could be removed altogether..

Reply:
Ok, good to get rid of it then